packages/core/admin/admin/src/content-manager/pages/App.tsx

```diff
@@ -123,8 +123,15 @@ const App = () => {
                 <ComponentSettingsView />
               </CheckPagePermissions>
             </Route>
+            <Redirect
+              from="/content-manager/collectionType/:slug"
+              to="/content-manager/collection-types/:slug"
+            />
```

Comment:
Do you think it could be beneficial adding a comment where why this exists?

Reply:
Yeah probably I'll do that when I fix my broken type.